frontend/src/shared/styles/theme.css

```
@@ -0,0 +1,38 @@
+    color: var(--color-slate-200);
+    margin: 0;
+    font-family: sans-serif;
+    
+  }
+}
+
+@theme {
+  /* 배경색 변수 추가 (기존 코드에서 빠졌던 부분) */
+  --color-bg: #0B1326;
+
+   /* 사이드 바, 메뉴 바 등등*/
```

Stylelint (17.5.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)